graphql-dxm-provider/src/main/java/org/jahia/modules/graphql/provider/dxm/nodetype/GqlJcrNodeType.java

```diff
@@ -199,6 +203,24 @@ public DXPaginatedData<GqlJcrNodeType> getNodeTypes(@GraphQLName("filter") @Grap
         return PaginationHelper.paginate(mapped, GqlJcrNodeType::getName, arguments);
     }
 
+    @GraphQLField
+    @GraphQLName("extendedBy")
+    @GraphQLDescription("Returns the node types dynamically extend by this node type, filtered by the specified parameters.")
```

Comment:
```suggestion
    @GraphQLDescription("Returns the node types that extend the current node type (in CND using the `extends=...` syntax), filtered by the specified parameters.")
```

Reply:
we use _this node type_ in the other description and not _the current node type_, we should probably be consistent